.github/workflows/parse_snyk_output.cjs

```diff
@@ -0,0 +1,117 @@
+const fs = require('fs');
+const JiraClient = require('jira-client');
+const axios = require('axios');
+
+
+console.log("JIRA_BASE_URL:", process.env.JIRA_BASE_URL);
+
+const jira = new JiraClient({
+  protocol: 'https',
+  host: process.env.JIRA_BASE_URL,
+  username: process.env.JIRA_USER_EMAIL,
+  password: process.env.JIRA_API_TOKEN,
+  apiVersion: '2',
+  strictSSL: true,
+});
+
+
+function parseSnykOutput(inputData) {
+  let vulnerabilities = [];
+  if (inputData) {
+    try {
+      const data = JSON.parse(inputData);
+      for (const project of data) {
+        vulnerabilities = vulnerabilities.concat(project.vulnerabilities);
+      }
+
+      console.log(vulnerabilities);
```

Comment:
Do we need to log vulnerabilities here? I'd prefer to have as little result logging as possible.

Reply:
Done!